### DISCUSSION TITLE
Individual Dice Metric for Each Label Segmentation in Auto3dseg

### DISCUSSION THREAD

**Discussion by Yasna-frn** [Project-MONAI/tutorials#1645] [General]
Can Auto3dseg provide separate Dice metrics for each label segmentation? This would enhance performance evaluation, especially for multi-label tasks. Any guidance on implementing this feature would be appreciated.

Thanks

**Comment by KumoLiu** [comment]
Hi @Yasna-frn, thanks for your interest here, you can set `return_with_label=True` in `MeanDice`, thanks.
https://github.com/Project-MONAI/MONAI/blob/ff430286c37e78d7592372a5a97377f0cbb0219c/monai/handlers/mean_dice.py#L54
